### DISCUSSION TITLE
Fundamentos do Blazor Web Assembly

### DISCUSSION THREAD

**Discussion by Mizaeldouglas** [balta-io/forum#543] [Tira Dúvidas] [answered]
a api que que menciona no video não esta disponível no Github:

`https://github.com/balta-io/2822/blob/main/src/encontro-premium-2023-05-04`

**Comment by andrebaltieri** [comment, marked as answer]
Segue o link:
https://github.com/balta-io/encontro-premium-2023-05-04
